Makefile

```diff
@@ -31,7 +33,18 @@ test:
 
 .PHONY: build
 build:
-	GOOS=$(GOOS) GOARCH=$(GOARCH) go build -o signoz-collector ./cmd/signozcollector
+	CGO_ENABLED=1 go build -tags timetzdata -o .build/${GOOS}-${GOARCH}/signoz-collector -ldflags "-linkmode external -extldflags '-static' -s -w ${LD_FLAGS}" ./cmd/signozcollector
+
+.PHONY: amd64
+amd64:
+	make GOARCH=amd64 build
+
+.PHONY: arm64
+arm64:
+	make CC=aarch64-linux-gnu-gcc GOARCH=arm64 build
```

Comment:
This requires the gcc installed on system and problem for working on any machine that doesn't use gnu gcc.

Reply:
yeah, you are right.

However, this is only meant to be used by us for cross-compiling for building multi-arch docker image.

For users wanting to build binary for their own machine, they can still use `make build`.